### DISCUSSION TITLE
About commits

### DISCUSSION THREAD

**Discussion by bash4Dev** [community/community#73513] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello everyone!
Oftentimes, when I pushed my contribution to the git, it will appear on the day before the exact day I pushed it on my git graph.

I mean, I push my commits today but they appear on my github as I push the commits yesterday. Please how can I troubleshoot this issue?

**Comment by b8zeek** [comment, marked as answer]
Greetings Bashir,

This is definitely related to the timezone settings on your Git or GitHub.

Check if your computer timezone is properly set, that might cause this issue. Check your GitHub timezone, I see yours seems right for Nigeria. Every commit has its own timestamp and it includes timezone so if you are using a service to commit for your, the commits might appear on the wrong day.

Check your commit with this command, it prints time and timezone: `git show --pretty=fuller`. See if it is in-sync, example SS below:
<img width="370" alt="Screenshot 2023-11-06 at 07 39 09" src="https://github.com/community/community/assets/24759563/077377e7-bf3e-4159-95c2-55a8a8912ab5">

Good luck! 🍺 

🍀

**Reply by bash4Dev** [reply]
Thank you so much, Mr Mirko!
I found this helpful and it really worked.

Goodluck too!

**Reply by b8zeek** [reply]
I am really glad I helped ser! ☺️

Enojoy the day!

🍀
